src/Extension/FluentVersionedExtension.php

```diff
@@ -168,4 +174,48 @@ protected function renameLocalisedTables(SQLSelect $query, array $tables)
             $query->renameTable($localisedTable, $localisedTable . '_' . self::SUFFIX_LIVE);
         }
     }
+
+    /**
+     * Apply versioning to write
+     *
+     * @param array $manipulation
+     */
+    public function augmentWrite(&$manipulation)
+    {
+        parent::augmentWrite($manipulation);
+
+        // Only rewrite if the locale is valid
+        $locale = $this->getWriteLocale();
+        if (!$locale) {
+            return;
+        }
+
+        // Get all tables to translate fields for, and their respective field names
+        $includedTables = $this->getLocalisedTables();
+        foreach ($includedTables as $table => $localisedFields) {
+            // Localise both _Versions and _Live writes
+            foreach ([self::SUFFIX_LIVE, self::SUFFIX_VERSIONS] as $suffix) {
+                $versionedTable = $table . '_' . $suffix;
+                $localisedTable = $this->getLocalisedTable($table) . '_' . $suffix;
```

Comment:
You _could_ pass suffix in, e.g. `getLocalisedTable($table, $suffix). The `$locale` argument is misleading in this context though - maybe we could rename it to `$extraSuffix` or something?